app/Filament/Pages/EditProfile.php

```diff
@@ -75,6 +78,7 @@ class EditProfile extends Page
 
     public ?array $data = [];
 
+    //TODO: I feel like a lot of these could be refactored into a settings file instead of adding them directly to the user migration.
```

Comment:
This makes sense, I'll create a ticket for us to explore moving somethings to some sort of User settings paradigm in the future.